src/features/scheduler/components/SchedulerMonthRows.tsx

```diff
@@ -91,32 +90,36 @@ export const SchedulerMonthRows = React.memo(
                         (event) => event.type === EventType.Holiday
                       )?.color || ""
                     }
-                    onClick={() =>
-                      onOpenCard?.({
+                    onClick={() => {
+                      return onOpenCard?.({
```

Comment:
why return?

Reply:
I have set console.log there)), I'll remove that